packages/xo-web/src/xo-app/vm/tab-console.js

```diff
@@ -38,15 +47,42 @@ export default class TabConsole extends Component {
     this.setState({ clipboard })
     this.refs.clipboard.value = clipboard
   }
+
+  _getClipboardContent = () => this.refs.clipboard && this.refs.clipboard.value
+
   _setRemoteClipboard = invoke(() => {
     const setRemoteClipboard = debounce(value => {
       this.setState({ clipboard: value })
       this.refs.noVnc.setClipboard(value)
     }, 200)
-    return event => setRemoteClipboard(event.target.value)
+    return value => setRemoteClipboard(value)
   })
 
-  _getClipboardContent = () => this.refs.clipboard && this.refs.clipboard.value
+  _setRemoteClipboardFromInput = event =>
+    this._setRemoteClipboard(event.target.value)
+
+  _showFormModal = async () =>
+    this._setRemoteClipboard(
+      await form({
+        defaultValue: this.state.clipboard,
+        header: (
+          <span>
+            <Icon icon='file-text' /> {_('copyToClipboard')}
```

Comment:
"Copy to clipboard" sounds like we're changing the value of the local clipboard.